audio-analysis/sqlConnection.py

```diff
@@ -0,0 +1,52 @@
+import pymysql
+from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String, Float, Numeric
+
+#### SQL CONNECTION INFO ####
+# Currently working on localhost
+# MAKE SURE YOU HAVE A DATABASE CALLED 'climactic_test'
+
+engine = create_engine('mysql+pymysql://root@localhost/climactic_test') # Make sure you use the correct username "root" and password
+
+#SQL Connection Setup - Checks if table exists and if URL to be input already exists
+def sqlConnectionSetup():
+    global engine
+    table_exists = False #Variable to check if table exists
+    # Create a metadata instance
+    meta = MetaData(engine)
+    #meta.reflect(bind=engine)
+
+    # Check if table already exists
+    for _t in meta.tables:
+        if _t == 'test_table':
+            table_exists = True
```

Comment:
You can add `return` here to stop the method from continuing. Then you won't need the `table_exists` variable

Reply:
Hey, could your clarify what you mean by 'return'?

Reply:
It would look like `if _t == 'test_table': return` When you call return the function will return nothing and the rest of the lines won't be executed.

Reply:
True, I'm stupid.. Lemme try that out

Reply:
Works and is cleaner